evals/cli/oaieval.py

```diff
@@ -140,7 +187,6 @@ def run(args: OaiEvalArguments, registry: Optional[Registry] = None) -> str:
     def parse_extra_eval_params(
         param_str: Optional[str],
     ) -> Mapping[str, Union[str, int, float]]:
-        """Parse a string of the form "key1=value1,key2=value2" into a dict."""
```

Comment:
Could you revert this changed line? This comment is useful since it specifies the format.